be/src/column/column_helper.h

```diff
@@ -132,6 +132,14 @@ class ColumnHelper {
 
         return dst_column;
     }
+    static ColumnPtr duplicate_column(ColumnPtr src, MutableColumnPtr desc, UInt32Column::Ptr offsets) {
+        auto desc_size = offsets->size() - 1;
+        DCHECK(src->size() >= desc_size) << "The size of the source column is less when duplicating it.";
+        for (int i = 0; i < offsets->size() - 1; ++i) {
+            desc->append_value_multiple_times(*src, i, offsets->get_data()[i + 1] - offsets->get_data()[i]);
+        }
+        return desc;
```

Comment:
can we leak the abstraction here? for an example:
In linear algebra, 
scalar * vector is not required to expanded to element-wise mutiplication of two vector. 
scalar * vector should outperform [scalar, scalar, scalar...] element-wise(*) vector.

Reply:
let it as future work, which should support a const op an array, based on consts column op non-const column.